shapely/tests/legacy/test_parallel_offset.py

```diff
@@ -30,21 +30,21 @@ def test_parallel_offset_linestring(self):
 
         line2 = LineString([(0, 0), (5, 0), (5, -5)])
         assert_geometries_equal(
-            line2.parallel_offset(2, "left", resolution=1),
+            line2.parallel_offset(2, "left", join_style=3),
             LineString([(0, 2), (5, 2), (7, 0), (7, -5)]),
         )
         assert_geometries_equal(
-            line2.parallel_offset(2, "left", join_style=2, resolution=1),
+            line2.parallel_offset(2, "left", join_style=2),
```

Comment:
The resolution keyword is ignored for a "mitre" join style, so just cleaning this up